examples/install/cluster-operator/04-Crd-kafka.yaml

```diff
@@ -1,16 +1,8 @@
-##---
-# Source: strimzi-kafka-operator/templates/04-Crd-kafka.yaml
```

Comment:
I thought these are still generated files? If so, there's no point changing them here.

If we're going to keep the generated CRD resources in git (it's only really useful for reviewing PRs) we also need an assertion in the CI that runs after they've been generated that checks that the version in git (i.e. which has been reviewed) is the same as the generated version. This sounds silly to me. I would opt for removing them from git altogether.

Reply:
As discussed in Slack - Tis PR is basicaly fixing them on master where they got out of sync with the Helm Chart PR and the listeners PR.

You already opened the PR with the check, so I assume this comment should be resolved.

Reply:
Yes, it's resolved by my PR